source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,124 @@
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Collapsed Reply Threads (General Availability)
+ - [Collapsed Reply Threads](XXXXXX link to CRT GA blog post) is now generally available. Please reference [this article](link to KB article XXXXXX) for more information and guidance for enabling the feature.
+
+#### Calls (Beta)
+ - Native voice calling and screen sharing is now available. This is a Channels-specific integration.
+
+#### Apps Bar (Beta)
+ - When a new configuration setting ``EnableAppBar`` is enabled, the channel header will be decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins will be moved to the new Apps Bar when the configuration setting is enabled. We recommend enabling the Apps Bar for servers with Calls enabled since the Apps Bar is the dedicated place for the Calls start/join button.
+
+#### Updated Server Hardware Requirements
+ - 
+
+#### Playbooks Updates
+ - Users can now easily keep processes up-to-date with the inline playbook editor.
+ - A new statistics dashboard was added that displays the number of playbooks and run instances within the server alongside other system statistics in the **System Console**.
+ - Run triggers and actions now provide more control over where status updates are posted throughout a run.
+
+#### Message Formatting Toolbar
+ - The [new formatting toolbar](https://docs.mattermost.com/channels/format-messages.html#use-the-messaging-formatting-toolbar) makes markdown accessible to everyone with easy to use controls for commonly used message formatting, such as bold, headings, links, and more.
+
+### Improvements
+
+#### User Interface (UI)
+ - For toggling the channel information in the right-hand pane, a shortcut CTRL/CMD+ALT+I was added.
+ - Added an "Unread Channels" section to the channel switcher and included threads in the results.
+ - To keep users in Mattermost when opening documentation links from the **System Console > Plugin** settings page, all the links now open in another tab.
+ - Users are no longer hidden from search results in the "Add members" modal, even if they are already members of the channel.
+ - Applied new designs for the Login screen:
+     - Default login
+     - OAuth options
+     - Custom branding
+     - MFA token
+     - Default signup
+     - OAuth options are available
+     - Custom branding is configured
+     - Email verification is enabled
+ - Changed **Actions** post menu hover text to **Message Actions**.
+ - Enabled the new onboarding task list for end users.
+ - The legacy ``Enable post formatting`` user setting is now hidden when the Message Formatting Toolbar is enabled.
+ - Updated Apps Framework to version 1.1.0 to add improved logging.
```

Comment:
There's a few improvements that might fit under an Integrations section

- To keep users in Mattermost when opening documentation links from the **System Console > Plugin** settings page, all the links now open in another tab.
- Changed **Actions** post menu hover text to **Message Actions**.
- Updated Apps Framework to version 1.1.0 to add improved logging.